docs/hub/mlx-image.md

```diff
@@ -0,0 +1,89 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering at the left of the [models page](https://huggingface.co/models?library=mlxim&sort=trending).
+There's also an open [mlx-vision](https://huggingface.co/mlx-vision) space for contributors converting and publishing weights for MLX format.
+
+Thanks to MLX Hugging Face Hub integration, you can load MLX models with a few lines of code. 
+
+## Installation
+
+```bash
+pip install mlx-image
```

Comment:
This downgraded several of my packages, including PyTorch, which went from 2.1.2 to 2.0.0. Not related to this PR, but is that necessary?

Reply:
No, it's not. I will fix it.

Reply:
Thank you! 🙌 We can still progress with the docs and release them :)